### PR TITLE
Added puzzles

### PR DESCRIPTION
**Pull request by reasonb4belief**
- New station: Quarantine (door and key based puzzles)
- New level on BioLab
- New station: Beta (for beta puzzles)
- New levels in Beta station: Orbital and Portal Mania

### CODE REVIEW

File: src/logic/puzzles.js [hunk 1]
```diff
@@ -241,6 +241,307 @@ export const puzzles = [
       features.outer,
     ],
   },
+  {
+    station: "Quarentine station",
```
Comment (skedwards88) [inline src/logic/puzzles.js:245]:
```suggestion
    station: "Quarantine station",
```

File: src/logic/puzzles.js [hunk 2]
```diff
@@ -241,6 +241,307 @@ export const puzzles = [
...
+    room: "1",
+    startingText:
+      "We're totally allowed to be in here! If you want to move quickly, try dragging along the path instead of tapping!",
+    winText: "The next one's a bit harder",
+    hintText: "Try grabbing the lower right key first",
```
Comment (skedwards88) [inline src/logic/puzzles.js:250]:
```suggestion
    winText: "The next one's a bit harder.",
    hintText: "Try grabbing the lower right key first.",
```

File: src/logic/puzzles.js [hunk 3]
```diff
@@ -241,6 +241,307 @@ export const puzzles = [
...
+      features.outer,
+      features.outer,
+      features.outer,
+      features.outer,
+      features.outer,
+      features.outer,
+    ],
+  },
+  {
+    station: "Quarentine station",
```
Comment (skedwards88) [inline src/logic/puzzles.js:320]:
```suggestion
    station: "Quarantine station",
```

File: src/logic/puzzles.js [hunk 4]
```diff
@@ -241,6 +241,307 @@ export const puzzles = [
...
+    room: "2",
+    startingText:
+      "I guess these keys are all that remians of subjects devoured by the slime... so sad",
```
Comment (skedwards88) [inline src/logic/puzzles.js:323]:
```suggestion
      "I guess these keys are all that remain of subjects devoured by the slime... So sad.",
```

File: src/logic/puzzles.js [hunk 5]
```diff
@@ -241,6 +241,307 @@ export const puzzles = [
...
+    winText: "We've almose broken out of quarentine, keep going!",
```
Comment (skedwards88) [inline src/logic/puzzles.js:324]:
```suggestion
    winText: "We've almost broken out of quarantine. Keep going!",
```

File: src/logic/puzzles.js [hunk 6]
```diff
@@ -241,6 +241,307 @@ export const puzzles = [
...
+    hintText: "Try saving the sample in the middle for the very end",
```
Comment (skedwards88) [inline src/logic/puzzles.js:325]:
```suggestion
    hintText: "Try saving the sample in the middle for the very end.",
```

File: src/logic/puzzles.js [hunk 7]
```diff
@@ -241,6 +241,307 @@ export const puzzles = [
...
+      features.outer,
+      features.outer,
+      features.outer,
+      features.outer,
+      features.outer,
+      features.outer,
+    ],
+  },
+  {
+    station: "Quarentine station",
```
Comment (skedwards88) [inline src/logic/puzzles.js:395]:
```suggestion
    station: "Quarantine station",
```

File: src/logic/puzzles.js [hunk 8]
```diff
@@ -241,6 +241,307 @@ export const puzzles = [
...
+    room: "3",
+    startingText:
+      "This one looks tricky. If you get stuck, you can backtrack or tap the start space to reset the room.",
+    winText: "Impressive... for a human",
```
Comment (skedwards88) [inline src/logic/puzzles.js:399]:
```suggestion
    winText: "Impressive...for a human.",
```

File: src/logic/puzzles.js [hunk 9]
```diff
@@ -241,6 +241,307 @@ export const puzzles = [
...
+    hintText: "Try grabbing the key above you, then immediately heading left.",
```
Comment (skedwards88) [inline src/logic/puzzles.js:400]:
```suggestion
    hintText: "Try grabbing the key above you first, then immediately heading left.",
```

File: src/logic/puzzles.js [hunk 10]
```diff
@@ -241,6 +241,307 @@ export const puzzles = [
...
+      features.outer,
+      features.outer,
+      features.outer,
+      features.outer,
+      features.outer,
+      features.outer,
+    ],
+  },
+  {
+    station: "Quarentine station",
```
Comment (skedwards88) [inline src/logic/puzzles.js:470]:
```suggestion
    station: "Quarantine station",
```

File: src/logic/puzzles.js [hunk 11]
```diff
@@ -241,6 +241,307 @@ export const puzzles = [
...
+    room: "4",
+    startingText: "Uh oh... do you think you can handle all these doors?",
```
Comment (skedwards88) [inline src/logic/puzzles.js:472]:
```suggestion
    startingText: "Uh oh...do you think you can handle all these doors?",
```

File: src/logic/puzzles.js [hunk 12]
```diff
@@ -241,6 +241,307 @@ export const puzzles = [
...
+    winText:
+      "Phew... we broke out of Quarentine. Time to explore the other stations. You can always revisit old rooms by tappng the map icon above.",
```
Comment (skedwards88) [inline src/logic/puzzles.js:474]:
```suggestion
      "Phew...we broke out of quarantine. Time to explore the other stations. You can always revisit old rooms by tapping the map icon above.",
```

File: src/logic/puzzles.js [hunk 13]
```diff
@@ -241,6 +241,307 @@ export const puzzles = [
...
+    hintText:
+      "Try leaving the key aobove you until you've crossed through a couple doors. Or tap next level to advance to the next station. You can always revisit old rooms by tappng the map icon above.",
```
Comment (skedwards88) [inline src/logic/puzzles.js:476]:
```suggestion
      "Try leaving the key above the start until you've crossed through a couple doors. Or tap next level to advance to the next station. You can always revisit old rooms by tapping the map icon above.",
```

File: src/logic/puzzles.js [hunk 14]
```diff
@@ -545,11 +846,85 @@ export const puzzles = [
...
+      features.outer,
+      features.outer,
+    ],
+  },
+  {
+    station: "Biolab station",
+    room: "6",
+    startingText: "XXX",
+    winText: "XXX",
+    hintText: "XXX",
```
Comment (skedwards88) [inline src/logic/puzzles.js:927]: needs text

File: src/logic/puzzles.js [hunk 15]
```diff
@@ -1824,4 +2199,229 @@ export const puzzles = [
...
+      features.outer,
+      features.outer,
+      features.outer,
+      features.outer,
+      features.outer,
+    ],
+  },
+  {
+    station: "Beta station",
+    room: "Protal Mania General",
```
Comment (skedwards88) [inline src/logic/puzzles.js:2279]:
```suggestion
    room: "Portal Mania General",
```

**Review by skedwards88 — COMMENTED** [review]
Looks good! Just some text updates.

File: src/logic/puzzles.js (hunk 14 above)
Comment (reasonb4belief) [reply src/logic/puzzles.js:927]:
```suggestion
    startingText: "Spry bottles, samples and doors, oh my!",
    winText: "Onward, to the Portal station!",
    hintText: "Try a spiral path and collecting the spray bottle partway through.",
```

**Review by skedwards88 — APPROVED** [review]

**Pull request merged**